nix/flakebox.nix

```diff
@@ -770,6 +777,11 @@ in
         "fedimint-dbtool"
         "fedimint-recoverytool"
       ];
+
+      preBuild = ''
+        cargo clean
+        export JEMALLOC_SYS_WITH_LG_PAGE=16
```

Comment:
Ok I have confirmed this works....now just need to figure out the correct spot for it